### PR TITLE
Add a guide for graphical artists

### PR DESCRIPTION
**Pull request by zenseii**
This adds a guide for people looking to help with the graphical assets of the project.

Relates to https://github.com/ihhub/fheroes2/issues/6553

~~This still needs the part about the palette to image tool https://github.com/ihhub/fheroes2/pull/6621 so this guide will need to be merged after that PR.~~

~~This will also need to be amended once this PR is merged: https://github.com/ihhub/fheroes2/pull/6626~~

~~I also need to add links to the main page etc.~~

~~As stated in the first mentioned issue, this guide needs to be included in the relevant archive. I suppose the precompiled tools?~~

### CODE REVIEW

File: docs/GRAPHICAL_ARTISTS.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# [**fheroes2**](README.md) graphical artist guide
+
+The original game has its graphical assets stored in the `HEROES2.agg` container and potentially the `HEROES2X.agg` container if the expansion
+pack Price of Loyalty is present. These two AGG containers are located in the `DATA` folder of the original Heroes 2 installation.
```
Comment (ihhub) [inline docs/GRAPHICAL_ARTISTS.md:4]: **The Price of Loyalty** to be more precise :)

File: docs/GRAPHICAL_ARTISTS.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# [**fheroes2**](README.md) graphical artist guide
+
+The original game has its graphical assets stored in the `HEROES2.agg` container and potentially the `HEROES2X.agg` container if the expansion
```
Comment (ihhub) [inline docs/GRAPHICAL_ARTISTS.md:3]: Shall we call files instead of containers to be simpler?

File: docs/GRAPHICAL_ARTISTS.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+pack Price of Loyalty is present. These two AGG containers are located in the `DATA` folder of the original Heroes 2 installation.
+
+For convenience's sake the fheroes2 project provides a set of tools and a script that artists can use to easily extract and convert the original
+graphical assets into viewable PNG image files.
```
Comment (ihhub) [inline docs/GRAPHICAL_ARTISTS.md:7]: We can remove **image** word as just state **PNG files**.

File: docs/GRAPHICAL_ARTISTS.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+New assets that the fheroes2 project adds to the game are included in the `files/data/resurrection.h2d` data file.
+
+This guide is designed for Linux, MacOS and Windows users. The source files to these tools are available in the project's repository.
+
+## Where to get the tools
+
+You can download the precompiled tools for your OS of choice here:
```
Comment (ihhub) [inline docs/GRAPHICAL_ARTISTS.md:15]: **operating system (OS)** just to avoid abbreviations without explanation.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @zenseii , I left several comments in this pull request. Could you please take a look at them when you have time?

**Review by oleg-derevenetz — APPROVED** [review]

**Comment by oleg-derevenetz** [comment]
> "GRAPHICAL_ARTISTS" does sound like it contains credits for the graphical artists of the project.

Maybe something like "GRAPHICAL_ASSETS" then?

File: docs/GRAPHICAL_ARTISTS.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+## Where to get the tools
+
+You can download the precompiled tools for your operating system (OS) of choice here:
+[Windows 64-bit](https://github.com/ihhub/fheroes2/releases/download/fheroes2-windows-x64-SDL2/fheroes2_tools_windows_x64_SDL2.zip)
+[Windows 32-bit](https://github.com/ihhub/fheroes2/releases/download/fheroes2-windows-x86-SDL2/fheroes2_tools_windows_x86_SDL2.zip)
+
+[MacOS](https://github.com/ihhub/fheroes2/releases/download/fheroes2-osx-sdl2_dev/fheroes2_tools_macos_x86-64_SDL2.zip)
+
+[Linux Ubuntu 64-bit](https://github.com/ihhub/fheroes2/releases/download/fheroes2-linux-sdl2_dev/fheroes2_tools_ubuntu_x86-64_SDL2.zip)
+[Linux Ubuntu ARM 64-bit](https://github.com/ihhub/fheroes2/releases/download/fheroes2-linux-arm-sdl2_dev/fheroes2_tools_ubuntu_arm64_SDL2.zip)
```
[the remark this answers is not in the document]
Comment (ihhub) [reply docs/GRAPHICAL_ARTISTS.md]: I have no clear opinion at the moment about including or excluding them from releases.

**Review by ihhub — APPROVED** [review]

File: docs/GRAPHICAL_ARTISTS.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ihhub) [reply docs/GRAPHICAL_ARTISTS.md]: If we decide to add the tools into the releases we cannot change these links as no releases contain the tools at the moment.

**Comment by ihhub** [comment]
@zenseii , huge thanks for tackling this feature!

**Pull request merged**